Assets/SC_Inentory.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Inentory : MonoBehaviour
+{
+    private bool _isFinish;
+
+    private bool salad;
+    private bool _bacon;
+    private bool CHEESE;
+
+    private void Awake()
+    {
+        GameEventSystem.OnPlayerPickUpCollectible += ASD;
+    }
+
+    private void OnDestroy()
+    {
+        GameEventSystem.OnPlayerPickUpCollectible -= ASD;
+    }
```

Comment:
There is no event system class in the repository. These lines will cause an error.